src/config.ts

```diff
@@ -9,6 +9,8 @@ export const indexFile = () => configuration().get<string>('baseRootFile') || 'i
 export const getBasePath = () => configuration().get<string>('basePath')
   || join(homedir(), 'vscode_wiki');
 
+export const getOpenRootFileFolder = () => configuration().get<boolean>('baseRootOpenAsFolder') || false;
+
```

Comment:
I think If there is no set value, it seems to have tried to get false as the default value.

Can we use this API?

![image](https://user-images.githubusercontent.com/14071105/103892647-42be1b80-512f-11eb-9cd8-4f45d0fb879b.png)

See also
  - https://code.visualstudio.com/api/references/vscode-api#WorkspaceConfiguration

Reply:
Sure, I'll correct it.